standards/github_folder_template.md

```diff
@@ -0,0 +1,50 @@
+# The .github folder and it's functionality
+
+### What is the .github folder?
+
+The `.github` folder allows users to store GitHub related files/folders. Some of these files and folders perform special functions for organizing your repo, issues, and PRs.
+
+Here are some of those functions:
+- Creating issue templates ([ISSUE_TEMPLATE folder](../.github/ISSUE_TEMPLATE))
+- Creating PR templates 
+- Creating a Code of Conduct
+- Gathering Funding for a project
+- Settings/config files for GitHub Actions
+- Security (reporting security vulnerabilities)
+- Code Owners (set default PR reviewers)
+
+Here is how the `.github` folder looks in this repo.
+![github folder](./assets/github_folder/1.png)
+
+### How to create an issue template
+Creating issue templates allows you to save time when creating issues. They are also beneficial for making sure assignees are including adequate information in the PR. In order to create an issue template, you must 
+1. Make sure you have a `.github` folder in your project's root directory
+2. Create a `ISSUE_TEMPLATE` folder in your `.github` folder
+3. Create a `.md` file in the `ISSUE_TEMPLATE` folder
+4. Add the following to the top of the file (change out this info with your own):
+![Issue template header](./assets/github_folder/2.png)
+_this is important because it lets GitHub know what kind of issue template you are creating_
+
+5. Use md to create the headers and body of the issue template. Such as the following:
+![Issue template](./assets/github_folder/3.png)
```

Comment:
I think instead of an image of the file in question, it would be better to link to the actual bug template and let the developer pull from that file.

Reply:
I like the idea, I will go ahead and make this change